### DISCUSSION TITLE
connect_log_param.py

### DISCUSSION THREAD

**Discussion by ramunozpr** [bitcraze/discussions#742] [Q&A]
Hi, sorry to bother but I'm new in this crazyflie software and I'm learning with the bitcraze forum but I seem to have a problem.
I'm doing the step-by-step connecting, logging and parameters tasks from the bitcraze forum and basically you have to create a python file in the crazy-lib-python folder and write a program where the crazyradio connects to the crazyflie for 3 seconds then disconnects, now I've done everything exactly how the forum says so far but when I execute the program as the forum tells me it doesn't execute anything, like there wasn't any code there, there are no reports of a problem, ive checked the code its exactly how the forum suggests, and I have all the per-requisites installed, and I still seem to not get out of this problem so if anybody knows whats going on please help me.

I will provide the code just in case needed:
```python
import logging
import time

import cflib.crtp
from cflib.crazyflie import Crazyflie
from cflib.crazyflie.syncCrazyflie import SyncCrazyflie

# URI to the Crazyflie to connect to
uri = 'radio://0/80/2M/E7E7E7E7E7'

def simple_connect():

    print("Yeah, I'm connected! :D")
    time.sleep(3)
    print("Now I will disconnect :'(")

if __name__ == '__main__':
    # Initialize the low-level drivers
    cflib.crtp.init_drivers()

    with SyncCrazyflie(uri, cf=Crazyflie(rw_cache='./cache')) as scf:

        simple_connect()
```
Also the link to the bitcraze website; https://www.bitcraze.io/documentation/repository/crazyflie-lib-python/master/user-guides/sbs_connect_log_param/

**Comment by krichardsson** [comment]
Hi!
Your code looks good I think, it rather sounds like something is wrong when you try to run it.
do you get any output at all in your terminal when you run `python3 connect_log_param.py`?
